src/content/smart-bulletins/035.md

```diff
@@ -0,0 +1,68 @@
+---
+title: "Smart Bulletin No. 035"
+intro: "GSA SmartPay® – OMB Circular No. A-123, Appendix B Agency Reporting to GSA/CCCM; Submission Implementing Guidance for Transit Benefit Accounts"
+number: "No. 035"
+order: 035
+businessline: "Purchase – accounts used for Transit Benefits"
+published: "Sep 1, 2020"
+lastupdated: "N/A"
+---
+
+UPDATE:
+| Date | Version History Action Log - Summary of Action/Changes |
+| ----------- | ----------- |
+| September 01, 2020 | Implementation of Smart Bulletin #35 |
+
+## Effective Date
+Effective upon issuance and shall remain in force until modified, rescinded, or superseded.
+
+## Business Line(s) Affected
+
+Purchase – accounts used for Transit Benefits
+
+## Introduction
+
+The purpose of this GSA SmartPay Smart Bulletin is to provide agencies/organizations that use the GSA SmartPay purchase business line for transit benefit accounts with 
```

Comment:
The purpose of this GSA SmartPay Smart Bulletin is to provide agencies/organizations that use the GSA SmartPay purchase business line for transit benefit accounts with implementing guidance on submitting statistical and narrative reports as required in Chapter 5.3 of Office of Management and Budget (OMB) Circular A-123, Appendix B, entitled “A Risk Framework for Government Charge Card Programs,” dated August 27, 2019. This Smart Bulletin includes: (1) reporting templates for agency submission of chapter 5.3 reports (2) reporting expectations for FY2020 and beyond.